.circleci/config.yml

```diff
@@ -4,65 +4,62 @@ orbs:
   aws-cli: circleci/aws-cli@0.1.13
   docker: circleci/docker@1.3.0
 
+platform_matrix: &platform_matrix
+  matrix:
+    parameters:
+      os: &oses ["linux", "darwin"] # <-- "windows" in the future!
+      arch: &arches ["amd64", "arm64"]
+    # exclude: # And when Windows comes, we'll need to exclude the Win+arm64 combo somehow
+    #   - os: "windows"
+    #   - arch: "arm64"
+
+jobs:
+  test:
+    docker:
+      - image: circleci/golang:1.16
+    steps:
+      - checkout
+      - run: go test --timeout 10s -v ./...
 
-commands:
   go-build:
+    docker:
+      - image: circleci/golang:1.16
     parameters:
       os:
         description: Target operating system
         type: enum
-        enum: ["linux", "darwin", "windows"]
+        enum: *oses
         default: "linux"
       arch:
         description: Target architecture
         type: enum
-        enum: ["amd64", "arm64"]
+        enum: *arches
         default: "amd64"
     steps:
+      - checkout
       - run: |
           GOOS=<< parameters.os >> \
           GOARCH=<< parameters.arch >> \
           go build -ldflags "-X main.BuildID=${CIRCLE_TAG}" \
-          -o $GOPATH/bin/honeytail-<< parameters.os >>-<< parameters.arch >> \
+          -o ~/artifacts/honeytail-<< parameters.os >>-<< parameters.arch >> \
           .
-
-jobs:
-  test:
-    docker:
-      - image: circleci/golang:1.13
-    steps:
-      - checkout
-      - run: go test --timeout 10s -v ./...
+      - persist_to_workspace:
```

Comment:
huh, are we expecting to see artifacts here: https://app.circleci.com/pipelines/github/honeycombio/honeytail/272/workflows/90c58c3d-4e1a-4d9c-8471-0a7132431256/jobs/843/artifacts

Reply:
I don't think so. This is passing some file system context between jobs during a workflow. To have things appear in artifacts, that's the `store_artifacts` command (used in our `build_packages`)

Reply:
I ... have yet to confirm that multiple concurrent jobs will be able to dump all their things in the same directory for `persist_to_workspace`. 😬  I aim to experiment with a `vx.y.z-rc1` tag to see them all come back together for the `build_packages` job after merge. 🤞🏻